### PR TITLE
DSS Infra: dev acm

### PR DESCRIPTION
**Pull request by amarjandu**
Update environment value for ACM.
Connected to #2536

### CODE REVIEW

File: environment.staging [hunk 1]
```diff
@@ -14,16 +14,11 @@ DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
 OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://${DCP_DOMAIN}/
 API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
 DSS_ZONE_NAME="staging.data.humancellatlas.org."
-ACM_CERTIFICATE_IDENTIFIER="f4f2b043-41ba-4116-b2f0-8d4296c3b97b"
+ACM_CERTIFICATE_IDENTIFIER="99fff90e-6ff5-44a5-852e-67c78f88c1f1"
 DSS_GCP_SERVICE_ACCOUNT_NAME="org-humancellatlas-staging"
 DSS_CHECKOUT_BUCKET_OBJECT_VIEWERS="serviceAccount:154609999906-compute@developer.gserviceaccount.com,serviceAccount:caas-account@broad-dsde-mint-staging.iam.gserviceaccount.com,serviceAccount:caas-prod-account-for-staging@broad-dsde-mint-staging.iam.gserviceaccount.com"
 DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE="org-humancellatlas-dss-{account_id}-${DSS_DEPLOYMENT_STAGE}-terraform"
 DSS_FLASHFLOOD_BUCKET=$DSS_FLASHFLOOD_BUCKET_STAGING
-AUTH_URL=https://auth.staging.data.humancellatlas.org
```
[the remark this answers is not in the document]
Comment (amarjandu) [reply environment.staging]: @chmreid good catch, resolved.

**Comment by chmreid** [comment]
Thanks Amar, also way to go increasing our code coverage by 30% :P

File: environment.staging [hunk 2]
```diff
@@ -14,7 +14,7 @@ DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
 OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://${DCP_DOMAIN}/
 API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
 DSS_ZONE_NAME="staging.data.humancellatlas.org."
-ACM_CERTIFICATE_IDENTIFIER="f4f2b043-41ba-4116-b2f0-8d4296c3b97b"
+ACM_CERTIFICATE_IDENTIFIER="99fff90e-6ff5-44a5-852e-67c78f88c1f1"
```
Comment (xbrianh) [inline environment.staging]: Is this addressed in  #2538 ?

Comment (amarjandu) [reply environment.staging]: closed #2538 proper SOP is merge into master and cherry pick into staging.

**Pull request merged**